elements/rh-button/rh-button.css

```diff
@@ -0,0 +1,391 @@
+  --_primary-focus-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+  --_primary-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_primary-hover-background-color: var(--rh-color-interactive-blue-darkest, #004080);
+
+  /* DANGER */
+  --_danger-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-background-color: #c9190b; /* WARNING: non-token value */
+  --_danger-border-color: transparent;
+  --_danger-active-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-active-background-color: #a30000; /* WARNING: non-token value */
+  --_danger-active-border-color: transparent;
+  --_danger-focus-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-focus-background-color: #a30000;  /* WARNING: non-token value */
+  --_danger-focus-border-color: transparent;
+  --_danger-hover-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_danger-hover-background-color: #a30000; /* WARNING: non-token value */
+  --_danger-hover-border-color: transparent;
+
+  /* SECONDARY */
+  --_secondary-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-background-color: transparent;
+  --_secondary-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-active-background-color: transparent;
+  --_secondary-active-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-focus-background-color: transparent;
+  --_secondary-focus-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_secondary-hover-background-color: transparent;
+  --_secondary-hover-border-color: var(--rh-color-interactive-blue-darker, #0066cc);
+
+  /* TERTIARY */
+  --_tertiary-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-background-color: transparent;
+  --_tertiary-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-active-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-active-background-color: transparent;
+  --_tertiary-active-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-focus-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-focus-background-color: transparent;
+  --_tertiary-focus-border-color: var(--rh-color-border-strong-on-light, #151515);
+  --_tertiary-hover-color: var(--rh-color-text-primary-on-light, #151515);
+  --_tertiary-hover-background-color: transparent;
+  --_tertiary-hover-border-color: var(--rh-color-border-strong-on-light, #151515);
+
+  /* LINK */
+  --_link-color: var(--rh-color-interactive-blue-darker, #0066cc);
+  --_link-background-color: transparent;
+  --_link-active-color: #a30000; /* WARNING: non-token value */
```

Comment:
Active:
Should be `--_link-active-color: var(--rh-color-interactive-blue-darkest, #004080);`

Hover:
`--_link-hover-color: var(--rh-color-interactive-blue-darkest, #004080);`

Focus:
`--_link-focus-color: var(--rh-color-interactive-blue-darkest, #004080);`
Blue border around button is correct.